doc/tsc-meetings/2015-07-22.md

```diff
@@ -0,0 +1,168 @@
+# Node.js Foundation TSC Meeting 2015-07-22
+
+## Links
+
+* **Audio Recording**: https://soundcloud.com/node-foundation/tsc-meeting-2015-07-22
+* **GitHub Issue**: https://github.com/nodejs/node-convergence-archive/issues/69
+* **Original Minutes Google Doc**: https://docs.google.com/document/d/1r8boI4E67Cq7PEsYeIpXkFZM0be4Ww5UDlNr_uXOop0
+
+## Agenda
+
+Extracted from **tsc-agenda** labelled issues and pull requests prior to meeting.
+
+### nodejs/io.js
+
+* doc: add GPG fingerprint for cjihrig [#2217](https://github.com/nodejs/io.js/pull/2217)
+* Process & Approval for Collab Summit Travel Fund [#2213](https://github.com/nodejs/io.js/issues/2213)
+* TC39 representation for the Node.js Foundation [#2153](https://github.com/nodejs/io.js/issues/2153)
+* Next branch release versioning [#2215](https://github.com/nodejs/io.js/issues/2215)
+
+## Minutes
+
+
+### Present
+
+* Mikeal Rogers
+* Rod Vagg (TSC)
+* Colin Ihrig (TSC)
+* James Snell (TSC)
+* Fedor Indutny (TSC)
+* Michael Dawson (TSC)
+* Steven R Loomis (TSC)
+* Jeremiah Senkpiel (TSC)
+* Brian White (TSC)
+* Ben Noordhuis (TSC)
+* Trevor Norris (TSC)
+* Chris Dickinson (TSC)
+* Mike Dolan (Linux F)
+* Emily Ratliff (Linux F)
+
+### Security Policy Discussion
+
+Emily Ratliff from the LF has joined us to help with our security and disclosure policy. TSC members were sent a briefing prior to the meeting.
```

Comment:
Ratliff is correct http://www.linuxfoundation.org/news-media/announcements/2015/06/linux-foundation-s-core-infrastructure-initiative-funds-three-new